extensions/ipynb/src/notebookAttachmentCleaner.ts

```diff
@@ -81,36 +81,31 @@ export class AttachmentCleaner implements vscode.CodeActionProvider {
 		this._disposables.push(vscode.workspace.onWillSaveNotebookDocument(e => {
 			if (e.reason === vscode.TextDocumentSaveReason.Manual) {
 				this._delayer.dispose();
-
-				e.waitUntil(new Promise((resolve) => {
-					if (e.notebook.getCells().length === 0) {
-						return;
+				if (e.notebook.getCells().length === 0) {
```

Comment:
Ignore this change, from another branch (part of https://github.com/microsoft/vscode/pull/207381#event-12081829987)